bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
#99 is now merged and it doesn't seem to be causing problems here so I think it's all good